### ISSUE TITLE
lx and ly not populated.

### ISSUE THREAD

**Issue by ValentineHerr** [SCBI-ForestGEO/2023census#65]
Hi @jess-shue,

I see a lot of missing lx and ly coordinates. I didn't think of checking for that in our QA/QC but I now see it.

Here are the number of cases with missing coordinates. Is there a way to retrieve them? I am hopeful that there is since the data is supposed to be georeferenced in the app..??

- old_tree --> 0 are missing corrdinates (all 49286 have coordinates)
- recruits_tree --> 5505 are missing coordinates (1 has a lx coordinate but missing ly)
- old_stem --> 5581 are missing coordinates (23270 gave coordinates)
- recruits_stem --> 1238 are missing coordinates (none have coordinates)

**Comment by jess-shue** [comment]
Hi @ValentineHerr,
   I will get back to you shortly on this - do not use the UTM coordinates just yet. When you download the data, a x and y column are included in decimal degrees from the app - the very last two columns on the old_trees and recruit_trees tables. We will need to convert these coordinates to calculate the lx, ly. Crews do not need to enter lx, ly in the field.

If an old tree was moved, these new coordinates will also capture those changes. So, the newly calculated lx, ly will be correct/changed for some stems. On my recent trip to Panama, I learned about this and also a new way of building a Notebook in the background that can be used to automatically calculate the new coordinates. Unfortunately, I did not know about this when setting things up for SCBI, but we can still make these coordinate calculations now.

Thanks,
Jess

**Comment by ValentineHerr** [comment]
Thanks for the clarification, @jess-shue. I am able to calculate local coordinates based on x and y.

For now, I am looking at the new NAD83 coordinates (calculated based on the decimal degree x, y) and am comparing them to the original coordinates in the plot below. There are trees that moved quite a bit. @Iriskennedy, @krystalbagnaschi and @rmh24 do you remember moving trees very far? (each blue line is an old tree moved)

My next step is to identify those trees and see if there is anything in the notes.

![image](https://github.com/SCBI-ForestGEO/2023census/assets/19673377/f2729faa-6713-4d55-a861-cbe9a3a54bbc)

**Comment by ValentineHerr** [comment]
Here are the 45 cases that have been moved more than 20m. Some have comments but others don't.

I find the little diagonal moves intriguing. Census crew, any idea why you would often have one tree move from one corner to the other?


@jess-shue, quadrat does not get edited when you move the tree?
 
![image](https://github.com/SCBI-ForestGEO/2023census/assets/19673377/dc5d7a26-b617-4657-b6ea-aadac9857ea4)

![image](https://github.com/SCBI-ForestGEO/2023census/assets/19673377/62906d08-e993-447f-8d46-ea9e1c8078fd)

**Comment by jess-shue** [comment]
@ValentineHerr No, again this is something we can do, but it was not built into SCBI's app.

**Comment by mitreds** [comment]
Hi Valentinne if you allow me, I will clarify your doubt about the movement of trees diagonally. This occurred in many cases because the person who mapped the trees in previous censuses was disoriented or had the maps upside down. The North behind them and the South towards their face. When normally they should map with the North in front and the South behind them.

So the trees' correct position would be the same, only on the opposite side of where they were mapped in previous censuses.

Atte,

**Comment by ValentineHerr** [comment]
@mitreds and @jess-shue thanks for the clarifications!
I also talked to the crew and we are trusting the new coordinates and change the lx, ly and quadrat accordingly.
